Makefile

```diff
@@ -212,17 +212,20 @@ format:
 ###                                Protobuf                                 ###
 ###############################################################################
 
+containerProtoVer=v0.7
```

Comment:
Might want to stick to "snake case" variable naming conventions :) variable_naming_conventions_are_fun :P

Reply:
changed